src/persist-txn/src/txns.rs

```diff
@@ -811,19 +812,19 @@ mod tests {
         // timestamp.
         let mut ts = 0;
         while ts < 32 {
-            info!("{} direct", ts);
             subs.push(txns.read_cache().expect_subscribe(&client, d0, ts));
             ts += 1;
+            info!("{} direct", ts);
```

Comment:
I kept having issues lining things up when looking into a bug I had during development. The new position of these logs means the number is the ts it actually happens at, which makes it _much_ easier